internal/v2/models/access_flow_to_model.go

```diff
@@ -142,6 +143,8 @@ func convertConditionToModel(ctx context.Context, condition client.ConditionPubl
 
 	if val, ok := condition.MatchOperator.Get(); ok {
 		model.MatchOperator = types.StringValue(val)
+	} else {
+		model.MatchOperator = types.StringValue("is") // Default match operator
```

Comment:
the API doesn't return this even though I'm sending it...
it also fails if I don't send it in the request. so setting it here explicitly.